crates/genesis/src/lib.rs

```diff
@@ -541,6 +545,21 @@ pub struct CliqueConfig {
     pub epoch: Option<u64>,
 }
 
+/// Consensus configuration for Parlia.
+/// Parlia is the consensus engine for BNB Smart Chain.
+/// For the general introduction: https://docs.bnbchain.org/docs/learn/consensus/
```

Comment:
doc nit again: 
for links you should follow recommendation here:

https://doc.rust-lang.org/rustdoc/lints.html?highlight=http%3A%2F%2F#bare_urls

Reply:
https://github.com/alloy-rs/alloy/actions/runs/9062315958/job/24951891283?pr=740

Reply:
fixed, thanks a lot